README.md

```diff
@@ -1,156 +1,74 @@
-      <td align="center" valign="top" width="14.28%"><a href="http://andortang.wordpress.com"><img src="https://avatars.githubusercontent.com/u/1508400?v=4?s=100" width="100px;" alt="Andy Tang"/><br /><sub><b>Andy Tang</b></sub></a><br /><a href="https://github.com/kadena-community/kadena.js/pulls?q=is%3Apr+reviewed-by%3AEnoF" title="Reviewed Pull Requests">👀</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/kate-hee-kyun-yun"><img src="https://avatars.githubusercontent.com/u/31594593?v=4?s=100" width="100px;" alt="Kate Hee Kyun Yun"/><br /><sub><b>Kate Hee Kyun Yun</b></sub></a><br /><a href="https://github.com/kadena-community/kadena.js/commits?author=ggobugi27" title="Code">💻</a></td>
-    </tr>
-    <tr>
-      <td align="center" valign="top" width="14.28%"><a href="http://react.cs.uni-sb.de/people/kuhtz.html"><img src="https://avatars.githubusercontent.com/u/1369810?v=4?s=100" width="100px;" alt="Lars Kuhtz"/><br /><sub><b>Lars Kuhtz</b></sub></a><br /><a href="https://github.com/kadena-community/kadena.js/commits?author=larskuhtz" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/nillo"><img src="https://avatars.githubusercontent.com/u/1943024?v=4?s=100" width="100px;" alt="de-nial-lo"/><br /><sub><b>de-nial-lo</b></sub></a><br /><a href="https://github.com/kadena-community/kadena.js/commits?author=nillo" title="Code">💻</a></td>
-    </tr>
-  </tbody>
-</table>
-
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
-
-<!-- ALL-CONTRIBUTORS-LIST:END -->
-
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+# Welcome to Kadena.js
+
+This monorepo contains libraries and tooling for frontend development.
+
+<picture>
+  <source srcset="./common/images/Kadena.JS_logo-white.png" media="(prefers-color-scheme: dark)"/>
+  <img src="./common/images/Kadena.JS_logo-black.png" width="200" alt="kadena.js logo" />
+</picture>
+
+## Support
+
+- [Kadena Documentation][1]
+- [Kadena on Discord][2]
+- [Kadena on Stack Overflow][3]
+
+## Contributing
+
+New contributors are welcome! If you're interested, please see [our guide to
+contributing][4].
+
+## Packages
+
+Overview of the main packages maintained in this repository:
+
+| Package                              | Release Notes        |
+| :----------------------------------- | :------------------- |
+| [@kadena/chainweb-node-client][5]    | [![version][7]][6]   |
+| [@kadena/chainweb-stream-client][8]  | [![version][10]][9]  |
+| [@kadena/chainwebjs][11]             | [![version][13]][12] |
+| [@kadena/client][14]                 | [![version][16]][15] |
+| [@kadena/cryptography-utils][17]     | [![version][19]][18] |
+| [@kadena/pactjs][20]                 | [![version][22]][21] |
+| [@kadena/pactjs-generator][23]       | [![version][25]][24] |
```

Comment:
I like this, do we want to add more info here. Similar to https://github.com/microsoft/rushstack#published-packages

Maybe we want to add the category that it belongs to? `apps` (or another name that represents deployables), `libs`, `tools`?

Reply:
Or have 3 sections with each their own list of packages?

Reply:
To be honest, one of my wishes is to eventually make `packages/*` a flat list, eliminating this kind of issues (and to get rid of semantics, e.g. "eslint-config" is not really a "tool"). I guess that's a discussion for another time.